docs/Developers/Development_Setup.rst

```diff
@@ -91,25 +91,21 @@ https://gitlab.com/coala/coala-utils/
 Installing from Git
 -------------------
 
-We recommend first installing the latest development snapshot of coala's
-master branch from and all of its dependencies with pip3 using
+We recommend installing coala from the master branch for latest updates
+and its dependencies with pip3 using
 
 ::
 
     (coala-venv)$ git clone https://github.com/coala/coala
     (coala-venv)$ cd coala
-    (coala-venv)$ pip3 install .
+    (coala-venv)$ pip3 install -e .
+    # This will install coala from the current directory.
```

Comment:
put this comment above the pip3 command, and instead of the imprecise `This ...` use `The next line ...`